### ISSUE TITLE
Suggestions for making  Portfolio Website better

### ISSUE THREAD

**Issue by vivekghanchi** [vivekghanchi/vivekghanchi.github.io#3]
Want some suggestion for the  betterment of my portfolio website

**Comment by engrravijain** [comment]
where should i update the suggestion and make a pull request ?

**Comment by AnaghSharma** [comment]
@vivekghanchi There are a lot of things that you can do. But first things first, try to answer these questions - 

1. Who is the owner of this project?
2. What do you want to achieve with the portfolio website.
3. Who is your audience? What value will you be bringing to different types of audience?

This all may seem like theoretical, but if you try to answer these questions and stick to them while designing it would help a lot.

**Comment by helloimkat** [comment]
Considering design - titles are too dark of a colour and blend in too much with the background. Also, you have different width containers and it's too hard to read on bigger screens. 

Would also be better to use maybe svg icons (font awesome or such) for your social links. 

If you want, I can play around and make a couple changes :)

**Comment by vivekghanchi** [comment]
@helloimkat  yes sure you can contribute to it. it would be a great help for me

**Comment by vivekghanchi** [comment]
@AnaghSharma  I am the owner of this project
I want to let people know my skills and qualifications and achievement through this site.

**Comment by vivekghanchi** [comment]
@engrravijain  u can pull request on its code

**Comment by AnaghSharma** [comment]
@vivekghanchi here go through [my website](https://anaghsharma.github.io) (a WIP) and see if you get what I was trying to convey.

Thanks.

**Comment by vivekghanchi** [comment]
@AnaghSharma Thankyou I want my portfolio to be like yours. a fully professionals

**Comment by xaiphron** [comment]
I'll send a PR for a sticky header to your repo. It works fine as it is but you can mod it if you want to.

**Comment by xaiphron** [comment]
I'll send a PR for a sticky header to your repo. It works fine as it is but you can mod it if you want to. Also, it has a JS script for smooth scroll.

**Comment by Paarmita** [comment]
The nav bar icons should be altogether ie. home is at other corner instead you could put a name logo there and home with other side. I could help in this if you want.

**Comment by Paarmita** [comment]
You could also add parallax to the page.

**Comment by AnaghSharma** [comment]
@vivekghanchi @Paarmita I suggest to not do anything like parallax just because it is cool and everyone is doing it. If you do not need it, do not use it. At this moment of time, focus on what your portfolio will be all about. Once the foundation has been set, you can try to enhance the visual design by adding unique elements (Not parallax, it is mainstream).

Thanks.

**Comment by hedelaerke** [comment]
Hi! If you will decide to keep your skill "progress bars" I do suggest making them animated. You can choose between little javascript implementation or try out CSS3 ::after pseudo-element.
